### PR TITLE
Gremlin dependencies

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

File: strongbox-db-server/pom.xml [hunk 1]
```diff
@@ -48,6 +48,53 @@
...
+                    <groupId>org.neo4j</groupId>
+                    <artifactId>neo4j-ogm-bolt-driver</artifactId>                    
+                </exclusion>
+            </exclusions>
+        </dependency-->
+        
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
```
Comment (carlspring) [inline strongbox-db-server/pom.xml:95]: This should be in the `strongbox-parent`, where I believe we already have a version defined for this.

Comment (sbespalov) [reply strongbox-db-server/pom.xml:95]: we only have `commons-lang3` there. Anyway, this is temporary soution to resolve OrientDB and Gremlin dependency issues and probably will be removed with Janusgraph.

File: pom.xml [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-60-SNAPSHOT</version>
     </parent>
     
     <artifactId>strongbox-db</artifactId>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-PR-16-SNAPSHOT</version>
```
Comment (steve-todorov) [inline pom.xml]: Version is incorrect - should be `18`
```suggestion
    <version>1.0-PR-18-SNAPSHOT</version>
```

File: strongbox-db-import/pom.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-db</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-16-SNAPSHOT</version>
```
Comment (steve-todorov) [inline strongbox-db-import/pom.xml]: Version is incorrect - should be `18`
```suggestion
        <version>1.0-PR-18-SNAPSHOT</version>
```

File: strongbox-db-liquibase/pom.xml [hunk 4]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-db</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-16-SNAPSHOT</version>
```
Comment (steve-todorov) [inline strongbox-db-liquibase/pom.xml]: Version is incorrect - should be `18`
```suggestion
        <version>1.0-PR-18-SNAPSHOT</version>
```

File: strongbox-db-server/pom.xml [hunk 5]
```diff
@@ -9,7 +9,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-db</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-16-SNAPSHOT</version>
```
Comment (steve-todorov) [inline strongbox-db-server/pom.xml]: Version is incorrect - should be `18`
```suggestion
        <version>1.0-PR-18-SNAPSHOT</version>
```

Comment (sbespalov) [reply strongbox-db-server/pom.xml]: fixed

File: strongbox-db-liquibase/pom.xml (hunk 4 above)
Comment (sbespalov) [reply strongbox-db-liquibase/pom.xml]: fixed

File: strongbox-db-import/pom.xml (hunk 3 above)
Comment (sbespalov) [reply strongbox-db-import/pom.xml]: fixed

File: pom.xml (hunk 2 above)
Comment (sbespalov) [reply pom.xml]: fixed

**Pull request closed without merging**